ThirdPartyModules/MageWorx/RewardPoints.php

```diff
@@ -0,0 +1,252 @@
+                if ($requestedPoints < 0.01) {
+                    // Reward Points amount is too small
+                    return [$discounts, $totalAmount, $diff];
+                }
+                $currencyAmount = $pointCurrencyConverter->getCurrencyByPoints(
+                    $requestedPoints
+                );
+                if (!$currencyAmount) {
+                    // Reward Points currency amount is zero
+                    return [$discounts, $totalAmount, $diff];
+                }
+                $currencyCode = $quote->getQuoteCurrencyCode();
+                $roundedAmount = CurrencyUtils::toMinor($currencyAmount, $currencyCode);
+                $discountType = $this->discountHelper->getBoltDiscountType('by_fixed');
+                $discounts[] = [
+                    'description'       => 'Reward Points',
+                    'amount'            => $roundedAmount,
+                    'reference'         => self::MAGEWORX_REWARD,
+                    'discount_category' => Discount::BOLT_DISCOUNT_CATEGORY_STORE_CREDIT,
+                    // For v1/discounts.code.apply and v2/cart.update
+                    'discount_type'     => $discountType,
+                    // For v1/merchant/order
+                    'type'              => $discountType,
+                ];
+                $diff -= CurrencyUtils::toMinorWithoutRounding($currencyAmount, $currencyCode) - $roundedAmount;
+                $totalAmount -= $roundedAmount;
+            }
+        } catch (\Exception $e) {
+            $this->bugsnagHelper->notifyException($e);
+        } finally {
+            return [$discounts, $totalAmount, $diff];
+        }
+    }
+
+    /**
+     * Add MageWorx Reward Points to layout to be rendered below the cart
+     *
+     * @param array $jsLayout
+     * @param \MageWorx\RewardPoints\Helper\Data $mageWorxRewardPointsHelper
+     *
+     * @return array
+     */
+    public function collectCartDiscountJsLayout(
+        $jsLayout,
+        $mageWorxRewardPointsHelper
+    ) {
+        if ($this->configHelper->getUseMageWorxRewardPointsConfig()
+            && $mageWorxRewardPointsHelper->isEnableForCustomer()) {
+            $jsLayout["mageworx-reward-points"] = [
+                "sortOrder" => 0,
+                "component" => "MageWorx_RewardPoints/js/view/payment/rewardpoints",
+                "children"  => [
+                    "errors" => [
+                        "sortOrder"   => 0,
+                        "component"   => "MageWorx_RewardPoints/js/view/payment/rewardpoints-messages",
+                        "displayArea" => "messages",
+                    ]
+                ]
+            ];
+            $jsLayout["mageworx-reward-points"]["config"] = [
+                "template" => $mageWorxRewardPointsHelper->isAllowedCustomPointsAmount()
+                            ? 'Bolt_Boltpay/third-party-modules/mageworx/reward-points/cart/rewardpoints_custom_amount',
```

Comment:
```suggestion
                            ? 'Bolt_Boltpay/third-party-modules/mageworx/reward-points/cart/rewardpoints_custom_amount'
```
this comma throws a syntax error, see the failing tests

Reply:
OH :( 

Fixed. Thankls